keyboards/inett_studio/sq80/info.json

```diff
@@ -0,0 +1,320 @@
+{
+    "keyboard_name": "SQUARE 80 %",
+    "manufacturer": "iNETT Studio",
+    "url": "https://geekhack.org/index.php?topic=100769.0",
+    "maintainer": "Wraul",
+    "usb": {
+        "vid": "0x694E",
+        "pid": "0x6060",
+        "device_version": "0.0.1"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "cols": [ "C7", "C6", "B6", "B3", "B4", "D7", "D6", "E6", "F0", "F1", "F4", "F5", "F7", "D5", "D3" ],
+        "rows": ["B0", "B1", "B2", "F6", "B5", "D4"]
+    },
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "D0",
+        "scroll_lock": "D1",
+        "num_lock": "D2",
+        "on_state": 0
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "community_layouts": ["tkl_ansi"],
+    "layouts": {
```

Comment:
looking at group buy info it appears there is a tsangan bottom row layout available also, essentially the `LAYOUT_tkl_ansi_wkl` layout is a wkl tsangan bottom row.
Would you be able to add a tsangan bottom row layout into `layouts` - named `LAYOUT_tkl_ansi_tsangan`?

Reply:
I'm sorry, now I'm a bit confused.
A Tsangan layout is supposed to have split backspace and right shift.
Or are you saying that the `LAYOUT_tkl_ansi_tsangan` isn't a true Tsangan layout and just refers to the bottom row? Which would be a bit weird.

Reply:
> are you saying that the LAYOUT_tkl_ansi_tsangan isn't a true Tsangan layout and just refers to the bottom row?

The QMK rendition/community_layout of a tenkeyless ANSI tsangan layout does include tsangan bottom row but not the split right shift and split backspace; a split bs and split right shift variant isn't currently available in QMK community layouts (`layouts/default/`) but I'm looking to add this variant in a respectively named PR.

Reply:
@Wraul 
Quickly picking up this bit - does this board support/have a tsangan bottom row layout? 🙂